homeassistant/helpers/config_validation.py

```diff
@@ -1683,7 +1706,9 @@ def _base_trigger_validator(value: Any) -> Any:
     return value
 
 
-TRIGGER_SCHEMA = vol.All(ensure_list, [_base_trigger_validator])
+TRIGGER_SCHEMA = vol.All(
+    ensure_list, [_base_trigger_validator, _backward_compat_trigger_schema]
```

Comment:
`[_base_trigger_validator, _backward_compat_trigger_schema]` means a list which passes at least one of `_base_trigger_validator, _backward_compat_trigger_schema`, that's not what we want.
```suggestion
    ensure_list, [vol.All(_base_trigger_validator, _backward_compat_trigger_schema)]
```

Reply:
ohhh, that makes sense